src/test/scala/mockws/Helpers.scala

```diff
@@ -0,0 +1,13 @@
+package mockws
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import play.api.mvc.{DefaultActionBuilder, PlayBodyParsers}
+import scala.concurrent.ExecutionContext.Implicits._
+
+object Helpers {
+  implicit val as = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  val bodyParser = PlayBodyParsers()
+  val action = DefaultActionBuilder(bodyParser.anyContent)
```

Comment:
We could also name it `Action`, as it's done in playframework: https://github.com/playframework/playframework/blob/master/framework/src/play/src/main/scala/play/api/mvc/Controller.scala#L161:
```
  def Action: ActionBuilder[Request, AnyContent] = controllerComponents.actionBuilder
```
WDYT?